api/auth.py

```diff
@@ -6,14 +6,21 @@
 """User authentication utilities"""
 
 from datetime import datetime, timedelta
-from typing import Optional
+from fastapi.security import OAuth2PasswordBearer
 from jose import JWTError, jwt
 from passlib.context import CryptContext
 from pydantic import BaseModel, BaseSettings, Field
 from .db import Database
 from .models import User
 
 
+user_scopes = {"admin": "Superusers", "users": "Regular users"}
+oauth2_scheme = OAuth2PasswordBearer(
+    tokenUrl="token",
+    scopes=user_scopes
+)
+
```

Comment:
The scopes could be part of the `Authentication` class. That would also probably solve the pylint error.

Reply:
Yes.